packages/client/hmi-client/src/page/project/components/tera-project-overview.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<tera-asset
 		:name="useProjects().activeProject.value?.name"
-		:authors="useProjects().activeProject.value?.authors?.join(', ')"
+		:authors="useProjects().activeProject.value?.authors?.sort().join(', ')"
```

Comment:
I hope that sorts by last name... 🤔

Reply:
Nope, it does not, I was messing around and forgot to change that, not really important to be honest.